test/scenarios.py

```diff
@@ -3943,348 +3943,348 @@
-                            event.Progress(0, FILES["nested/big/testfile-02"].id),
-                        ]
-                    ),
-                    action.CancelTransferFile(0, FILES["nested/big/testfile-01"].id),
-                    action.CancelTransferFile(0, FILES["nested/big/testfile-02"].id),
-                    action.WaitRacy(
-                        [
-                            event.FinishFileCanceled(
-                                0, FILES["nested/big/testfile-01"].id, False
-                            ),
-                            event.FinishFileCanceled(
-                                0, FILES["nested/big/testfile-02"].id, False
-                            ),
-                        ]
-                    ),
-                    action.Download(
-                        0,
-                        FILES["nested/big/testfile-01"].id,
-                        "/tmp/received/21-3",
-                    ),
-                    action.WaitForResume(
-                        0,
-                        FILES["nested/big/testfile-01"].id,
-                        f"/tmp/received/21-3/*{FILES['nested/big/testfile-01'].id}.dropdl-part",
-                    ),
-                    action.Wait(
-                        event.FinishFileDownloaded(
-                            0,
-                            FILES["nested/big/testfile-01"].id,
-                            "/tmp/received/21-3/nested/big/testfile-01",
-                        )
-                    ),
-                    action.Download(
-                        0,
-                        FILES["nested/big/testfile-02"].id,
-                        "/tmp/received/21-3",
-                    ),
-                    action.WaitForResume(
-                        0,
-                        FILES["nested/big/testfile-02"].id,
-                        f"/tmp/received/21-3/*{FILES['nested/big/testfile-02'].id}.dropdl-part",
-                    ),
-                    action.Wait(
-                        event.FinishFileDownloaded(
-                            0,
-                            FILES["nested/big/testfile-02"].id,
-                            "/tmp/received/21-3/nested/big/testfile-02",
-                        )
-                    ),
-                    action.CheckDownloadedFiles(
-                        [
-                            action.File(
-                                "/tmp/received/21-3/nested/big/testfile-01", 10485760
-                            ),
-                            action.File(
-                                "/tmp/received/21-3/nested/big/testfile-02", 10485760
-                            ),
-                        ],
-                    ),
-                    action.CancelTransferRequest(1),
-                    action.ExpectCancel([1], False),
-                    action.NoEvent(),
-                    action.Stop(),
-                ]
-            ),
-        },
-    ),
+    # Scenario(
```

Comment:
Let's leave a TODO comment here on why we disable it and when it should be enabled